### DISCUSSION TITLE
Cloud Media App in Android

### DISCUSSION THREAD

**Discussion by claytondecker** [ente-io/ente#1266] [Enhancements]
Should Ente be an option as a cloud media app in Android?

![cloud-media-app](https://github.com/ente-io/ente/assets/112224219/5c21bfd2-28d1-4d98-9f55-eaa0ab0fd446)

**Comment by vishnukvmd** [comment]
Please note:
>  Only apps nominated by OEMs are eligible to participate in this program to become a cloud media provider for Android at this time.

Source: https://developer.android.com/guide/topics/providers/cloud-media-provider#eligibility

**Reply by ItsHarper** [reply]
That's true for devices running Google-certified Android. I've opened https://github.com/GrapheneOS/os-issue-tracker/issues/6176, and I'm hoping that a future version of GrapheneOS will allow any app that implements the API to be selected.
